dev/12_optimizer.ipynb

```diff
@@ -587,7 +587,7 @@
     "state = average_grad(state, p,  mom=0.9, dampening=True)\n",
```

Comment:
<p>There is no need to import partial here, it's in the basics imports of fastai so already in the state.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
Oh yep, will remove now